### DISCUSSION TITLE
Writing to plaintext?

### DISCUSSION THREAD

**Discussion by mike-lawrence** [zarr-developers/zarr-python#1063] [General]
When debugging projects using zarr, it would be convenient to be able to use my OS's file explorer to preview the content of datasets as plaintext of some sort (ex. csv). None of the storage formats achieve this, right?

**Comment by sanketverma1704** [comment]
Hi @mike-lawrence.
This seems more like a discussion topic to me rather than an issue.

Maybe @joshmoore, @jakirkham or @rabernat can help here?

**Reply by joshmoore** [reply]
In fact, @mike-lawrence, I'm not sure what you mean. Are you saying that you would like to see the _metadata_ as a tree that you can explore or the _binary_ data? Do you have an example of what your Zarr fileset looks like?

**Comment by jakirkham** [comment]
Not sure exactly what is being asked for here, but would [`Group.tree()`]( https://zarr.readthedocs.io/en/stable/api/hierarchy.html#zarr.hierarchy.Group.tree ) help in exploring this hierarchy? This shows the hierarchy in plain text. Also in Jupyter it uses `ipytree` to provide an interactive Jupyter widget.
